docs/README-develop-beaker-scripts.md

```diff
@@ -118,7 +142,7 @@ HOSTS:
           user: <configured admin username>
           password: <configured admin password>
 
-    <Nexus agent>:
+    <Nexus bash-shell or guestshell agent>:
```

Comment:
I think this should be commented out, ie:

`#    <Nexus bash-shell or guestshell agent>: `